caiman/summary_images.py

```diff
@@ -694,20 +694,30 @@ def local_correlations_movie(file_name,
 
 def local_correlations_movie_offline(file_name,
                                      Tot_frames=None,
-                                     fr: int = 10,
+                                     fr: float = 10.,
                                      window: int = 30,
```

Comment:
The `window, stride, winSize_baseline` have very low default values. Possibly change.